comp/core/tagger/common/datadog_config.go

```diff
@@ -0,0 +1,14 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+// Package common provides common utilities that are useful when interacting with the tagger.
+package common
+
+// DatadogConfig contains the configuration specific to Dogstatsd.
+type DatadogConfig struct {
+	DogstatsdEntityIDPrecedenceEnabled bool
+	DogstatsdOptOutEnabled             bool
+	OriginDetectionUnifiedEnabled      bool
```

Comment:
Could we add docstring for those to document what they enable ?

Reply:
Done https://github.com/DataDog/datadog-agent/pull/30584/commits/b199f580e2e55e558b63509e94f34cd624eadf6e